vtkext/private/module/vtkF3DEXRReader.cxx

```diff
@@ -8,11 +8,68 @@
 #include "vtksys/FStream.hxx"
 
 #include <ImfArray.h>
+#include <ImfIO.h>
 #include <ImfRgbaFile.h>
 
+#include <cstring>
 #include <sstream>
 #include <thread>
 
+/**
+ * Class to treat file contents in memory like it were still in a file.
+ */
+class MemStream : public Imf::IStream
+{
+public:
+  MemStream(const char* name, const void* buff, vtkIdType bufferLen)
+    : Imf::IStream(name)
+    , buffer(static_cast<const char*>(buff))
+    , bufflen(static_cast<size_t>(bufferLen))
+    , pos(0)
+  {
+  }
+
+  bool read(char c[], int n) override
+  {
+    if (pos + n <= bufflen)
+    {
+      memcpy(c, buffer + pos, n);
+      pos += n;
+      return true;
+    }
+    return false;
+  }
+
+  /**
+   * returns the current reading position, in bytes, from the beginning of the file.
+   * The next read() call will begin reading at the indicated position
+   */
+  uint64_t tellg() override
+  {
+    return pos;
+  }
+
+  /**
+   * sets the current reading position to pos bytes from the beginning of the "file"
+   */
+  void seekg(uint64_t new_pos) override
+  {
+    pos = new_pos;
+  }
+
+  /**
+   * clears any error flags (we dont have to worry about this)
+   */
+  void clear() override
```

Comment:
if not used, not sure we need to define it.

Reply:
correct this is not needed, removing